overcast/django/apps/buildsvc/pkgbuild/__init__.py

```diff
@@ -13,12 +14,14 @@
 from ..models import BuildRecord
 
 class PackageBuilder(object):
-    def __init__(self, basedir, package_source, build_record):
+    def __init__(self, basedir, package_source, build_record,
+                 docker_build=True):
```

Comment:
Noone calls PackageBuilder directly. It's only ever instantiated in one place and it doesn't pass a value for that keyword arg. There is no way to make use of these code branches.

Come up with a use case. If you can do that, make it configurable. Right now, none of us seem able to come up with a use case for sbuild-based builds that we care about and even if we had one, we couldn't even configure the build service to us sbuild.